tests_pytest/pytorch_tests/integration_tests/core/quantization/test_set_node_quantization_config.py

```diff
@@ -0,0 +1,128 @@
+            mx_cfg_list.append(base_cfg.clone_and_edit(attr_to_edit={BIAS_ATTR: {WEIGHTS_N_BITS: n}}))
+        mx_cfg_list.append(
+            base_cfg.clone_and_edit(attr_to_edit={KERNEL_ATTR: {WEIGHTS_N_BITS: 4}, BIAS_ATTR: {WEIGHTS_N_BITS: 16}})
+        )
+        tpc = generate_tpc(default_config=default_config, base_config=base_cfg, mixed_precision_cfg_list=mx_cfg_list,
+                           name='imx500_tpc_kai')
+
+        return tpc
+
+    def representative_data_gen(self, shape=(3, 8, 8), num_inputs=1, batch_size=2, num_iter=1):
+        for _ in range(num_iter):
+            yield [torch.randn(batch_size, *shape)] * num_inputs
+
+    def get_float_model(self):
+        class BaseModel(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv1 = torch.nn.Conv2d(in_channels=3, out_channels=3, kernel_size=3)
+                self.conv2 = torch.nn.Conv2d(in_channels=3, out_channels=3, kernel_size=3)
+                self.relu = torch.nn.ReLU()
+
+            def forward(self, x):
+                x = self.conv1(x)
+                x = self.conv2(x)
+                x = self.relu(x)
+                return x
+        return BaseModel()
+
+    def get_test_graph(self, core_config):
+        float_model = self.get_float_model()
+        fw_info = DEFAULT_PYTORCH_INFO
+
+        fw_impl = PytorchImplementation()
+        graph = fw_impl.model_reader(float_model,
+                                     self.representative_data_gen)
+        graph.set_fw_info(fw_info)
+
+        tpc = self.get_tpc()
+        attach2pytorch = AttachTpcToPytorch()
+        fqc = attach2pytorch.attach(
+            tpc, core_config.quantization_config.custom_tpc_opset_to_layer)
+        graph.set_fqc(fqc)
+
+        return graph
+
+    # test case for set_manual_activation_bit_width
+    """
+    Test Items Policy:
+        - How to specify the target layer: Options(type/name)
+        - Target attribute information: Options(kernel/bias) 
+        - Bit width variations: Options(2, 4, 16)
+    """
+    test_input_1 = (NodeNameFilter("conv1"), 2, PYTORCH_KERNEL)
+    test_input_2 = (NodeTypeFilter(nn.Conv2d), 16, PYTORCH_KERNEL)
+    test_input_3 = ([NodeNameFilter("conv1"), NodeNameFilter("conv1")], [4, 16], [PYTORCH_KERNEL, BIAS])
+
+    test_expected_1 = ({"conv1": {PYTORCH_KERNEL: 2, BIAS: 32}, "conv2": {PYTORCH_KERNEL: 8, BIAS: 32}})
+    test_expected_2 = ({"conv1": {PYTORCH_KERNEL: 16, BIAS: 32}, "conv2": {PYTORCH_KERNEL: 16, BIAS: 32}})
+    test_expected_3 = ({"conv1": {PYTORCH_KERNEL: 4, BIAS: 16}, "conv2": {PYTORCH_KERNEL: 8, BIAS: 32}})
+
+    @pytest.mark.parametrize(
+        ("inputs", "expected"), [
+        (test_input_1, test_expected_1),
+        (test_input_2, test_expected_2),
+        (test_input_3, test_expected_3),
+    ])
+    def test_manual_weights_bitwidth_selection(self, inputs, expected):
```

Comment:
Please also add a test for manual selection bits for positional attributes (constant inputs of a functional node like 'Add')